packages/templates/app/typescript/src/ethereum.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseEthereum } from "./wrap";
+
+import { CoreClient, PolywrapClient } from "@polywrap/client-js";
+
+export class Ethereum extends BaseEthereum {
```

Comment:
As an app developer, it feels weird that I have to do this every time. I would imagine that the code below would be the "sane defaults", and if I wanted to change it I could do something like:
```typescript
import { Ethereum as DefaultEthereum } from "./wrap"

class Ethereum extends DefaultEthereum {
  protected _getDefaultClient(): CoreClient {
    // TODO: create your own custom client
  }
}
```